CHANGELOG.md

```diff
@@ -67,6 +67,12 @@ and this project adheres to
   lexicographical ordering as implemented by `Vec<u8>`.
 - cosmwasm-std: Implement `PartialOrd` and `Ord` for `Addr` using the same
   lexicographical ordering as implemented by `String`.
+- cosmwasm-std: Added new `WasmMsg::UpdateAdmin` variant that allows an admin
+  contract (eg. multisig) to set another admin ([#900])
+- cosmwasm-std: Added new `WasmMsg::UpdateAdmin` variant that allows an admin
+  contract (eg. multisig) to set another admin ([#900])
+- cosmwasm-std: Add optional `admin` field to `WasmMsg::Instantiate` to allow a
+  contract to be full admin (able to migrate) a contract it created
```

Comment:
This is already covered in the Changed block (see line 113-116 before your addition)

Reply:
I missed that